config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/tasks/zip/builder/IdentityProviderEntityBuilder.java

```diff
@@ -16,10 +17,12 @@
 import org.w3c.dom.Element;
 
 import java.util.List;
-import java.util.Map;
 import java.util.stream.Collectors;
 
+import static com.ca.apim.gateway.cagatewayconfig.util.gateway.BundleElementNames.*;
+
 public class IdentityProviderEntityBuilder implements EntityBuilder {
+    private static final String TRUSTED_CERT_URI = "http://ns.l7tech.com/2010/04/gateway-management/trustedCertificates";
```

Comment:
should we create a class for URI constants?

Reply:
maybe if we continue to add more URIs? for now it might be ok to just put it where it's used